helm/hades/crds/build.hades.tum.de_buildjobs.yaml

```diff
@@ -0,0 +1,144 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.18.0
+  name: buildjobs.build.hades.tum.de
+spec:
+  group: build.hades.tum.de
+  names:
+    kind: BuildJob
+    listKind: BuildJobList
+    plural: buildjobs
+    singular: buildjob
+  scope: Namespaced
+  versions:
+    - name: v1
+      schema:
+        openAPIV3Schema:
+          description: BuildJob is the Schema for the buildjobs API.
```

Comment:
I think this description should be improved. I don't think people would understand if the dont know hades very well